asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -100,6 +134,49 @@ def write_html(html, path):
         with open(path, "w") as f:
             f.write(html)
 
+    def swap_b_factor(self, protein, fitness_data):
```

Comment:
We could always do the following but I think we can leave it for now in thiis PR. https://www.mdanalysis.org/MDAnalysisTutorial/writing.html#example-saving-dynamic-per-atom-properties-in-b-factor

Reply:
No longer relevant.